app/config/locale/continents.php

```diff
@@ -1,11 +1,46 @@
 <?php
 
+/**
```

Comment:
Might be nice to also mention for what purposes coordinates are, I am guessing to recommend best region.